website/blog/2024-01-24-release-1.7.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Release Notes - Podman Desktop 1.7
+description: Podman Desktop 1.7 has been released!
+slug: podman-desktop-release-1.7
+authors: deboer
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.7/renovations.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.7 Release! 🎉
+
+![Podman-desktop-1-7-hero](img/podman-desktop-release-1.7/renovations.png)
+
+This release introduces:
+
+- **Podman 4.9.0**: [Podman 4.9.0](https://github.com/containers/podman/releases) is now included in Windows and Mac installers.
+- **Extension API Improvements**: Another set of improvements to the extension API enabling more goodness for 🦭 Podman Desktop's extensions.
+- **Experimental Kubernetes UI**: 
+
+Podman Desktop 1.7 is now available. [Click here to download it](/downloads)!
+
+---
+
+## Release Details
+
+### Extension API improvements
+
+The 🦭 Podman Desktop extension API received many improvements, including:
```

Comment:
... received many ex-seal-ent improvements, including:

Reply:
Text had changed from the draft so I didn't include this one.